scripts/start-multi-node-test.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+
+# Quality set for bash scripts:
+# [set -e] fails on error
+set -e
+
+# [set -u ] stops the script on unset variables
+set -u
+
+# [set -o pipefail] triggers a failure if one of the commands in the pipe fail
+set -o pipefail
+
+node=$1
+shift
+partitions=$#
+relative_configs=$@
+
+if [[  $partitions < 2 ]]
+then
+    echo "Syntax: ./scripts/start-multi-node-test.sh node-number config1 config2 [config3 [config4 ...]]"
+    exit 1
+fi
+
+if [ ! -d scripts ]
+then
+    echo "This script must be launched at the top of the repository, as ./scripts/start-multi-node-test.sh"
+    exit 1
+fi
+
+multi_dir="split$partitions"
+
+if [ -d "$multi_dir" ]
+then
+    echo "directory '$multi_dir' already exists"
+    exit 1
+fi
+
+function exists_in_path {
+    what=$1
+    for dir in $(echo $PATH | tr ':' ' ')
+    do
+        wanted=$dir/$what
+        if [ -x $wanted ]
+        then
+            echo $wanted
+            return
+        fi
+    done
+}
+
+for needed in tmux git realpath jq
```

Comment:
Are `jq` or `tmux` needed here? I don't see them being used in the code.

Reply:
same as previous one